GMCPagingScrollView/GMCPagingScrollView.m

```diff
@@ -152,31 +152,31 @@ - (CGRect)frameForPageAtActualIndex:(NSUInteger)index {
 }
 
 - (void)layoutSubviews {
-	[super layoutSubviews];
-	
-	CGRect frameForScrollView = [self frameForScrollView];
-	if (!CGRectEqualToRect(self.scrollView.frame, frameForScrollView)) {
+    self.inLayoutSubviews = YES;
```

Comment:
Have you tested this on iOS 7 and 8, with different orientations and starting from different places? This impacts what happens in `scrollViewDidScroll:`. With this change, it may not bring in some pages initially. I'd suggest testing on later sections in D&C, for example, on iOS 6, 7, and 8, in both orientations.